test/pan-and-zoom.spec.ts

```diff
@@ -1,6 +1,5 @@
-"use strict";
-
-const PanAndZoom = require("../pan-and-zoom.js");
+/* eslint-disable no-undef */
```

Comment:
Why was this added?

Reply:
ESLint thinks that 'expect' is not defined

Reply:
Oh, right. You've added an `.eslintrc.js` file, which shadows the [`"eslintConfig"`](https://github.com/Alhadis/PanAndZoom/blob/00d5cdc0ca2ae7d99d135d1441f7fb6cf18646d6/package.json#L26-L35) field in `package.json`. You should merge the contents of that entry with your `.eslintrc.js`: it defines `expect` as a global variable for ESLint so it won't complain about it.